client/my-sites/email/google-sale-banner/index.tsx

```diff
@@ -56,6 +58,10 @@ const GoogleSaleBanner: FunctionComponent< GoogleSaleBannerProps > = ( { domains
 		domainForSale?.blogId ? getSite( state, domainForSale.blogId ) : getSelectedSite( state )
 	);
 
+	if ( isMobile ) {
+		return null;
```

Comment:
Curious: Why did you switch to a no-style solution?

Reply:
I wanted to be sure that we wouldn't trigger the Tracks events -- I suspected that the `TrackComponentView` logic in the `Banner` component would still trigger an event even if the CSS was set to `display: none;`. So I want to be extra-sure we were tracking things correctly.